mod/primitives/pkg/bytes/b.go

```diff
@@ -50,6 +50,6 @@ func (b *Bytes) UnmarshalText(input []byte) error {
 }
 
 // String returns the hex encoding of b.
-func (b Bytes) String() hex.String {
-	return hex.FromBytes(b)
+func (b Bytes) String() string {
```

Comment:
with this change hex.String is type is called in `string.go` and `hex_test.go` only